hmt_escrow/eth_bridge.py

```diff
@@ -77,13 +77,20 @@ def get_w3() -> Web3:
     <class 'web3.providers.websocket.WebsocketProvider'>
     >>> del os.environ["HMT_ETH_SERVER"]
 
+    Args:
+        hmt_server_addr: infura API address.
+
     Returns:
         Web3: returns the web3 provider.
 
     """
-    endpoint = os.getenv("HMT_ETH_SERVER", "http://localhost:8545")
+    endpoint = None
+
+    if hmt_server_addr:
+        endpoint = hmt_server_addr
+
     if not endpoint:
-        LOG.error("Using EthereumTesterProvider as we have no HMT_ETH_SERVER")
```

Comment:
This removes error logging, in case where the HMT_ETH_SERVER env variable is defined and is "" (empty string). If it's accidental, might be better to return it back.